README.md

```diff
@@ -35,9 +35,9 @@ public protocol SonarViewDataSource: class {
 
 `SonarItemView` is just a UIView subclass. In order to use your custom view in radar, make your view SonarItemView subclass. The size of view is determined via layout. The layout is defined by SonarViewLayout protocol. Sonar comes with one predefined layout - SonarViewCenteredLayout. The items in a wave are placed from edges to the center, for example if we have 4 items in wave, first is placed at the left side, second at the right side, third at the left again but a right from the first one and so on. I know, the image would be better:
 
-<img src="https://raw.githubusercontent.com/thefuntasty/Sonar/master/sonarLayoutScreenshot.png" width="300" />
+<img src="https://raw.githubusercontent.com/futuredapp/Sonar/master/sonarLayoutScreenshot.png" width="300" />
 
-If you need different behaviour, you can of course create your own layout by adopting the SonarViewLayout a pass it to the SonarView. See [SonarViewLayout](https://github.com/thefuntasty/Sonar/blob/master/Pod/Classes/SonarViewLayout.swift) for more information.
+If you need different behaviour, you can of course create your own layout by adopting the SonarViewLayout a pass it to the SonarView. See [SonarViewLayout](https://github.com/futuredapp/Sonar/blob/master/Pod/Classes/SonarViewLayout.swift) for more information.
```

Comment:
```suggestion
If you need different behaviour, you can of course create your own layout by adopting the SonarViewLayout and pass it to the SonarView. See [SonarViewLayout](https://github.com/futuredapp/Sonar/blob/master/Pod/Classes/SonarViewLayout.swift) for more informations.
```

Reply:
Nehleď prosím tolik na gramatiku, když to není ve scope PR a nejseš si 100% jistý, že je to správně. Information je nepočitatatelné, stejně jako v češtině.

Reply:
Byla to jenom suggestion jak to trochu vylepsit kdyz uz jsi u tech zmen. Mas pravdu. Informations neexistuje. 😢